javascripts/tagmanagerHelper.js

```diff
@@ -40,11 +40,10 @@
     };
 
     tagManagerHelper.createNewVersion = function () {
-        var piwikUrl = piwikHelper.getAngularDependency('piwikUrl');
-        var containerId = piwikUrl.getSearchParam('idContainer');
+        var containerId = CoreHome.MatomoUrl.parsed.value.idContainer;
         this.editVersion(null, containerId, 0, function () { window.location.reload(); });
     };
-    tagManagerHelper.editVersion = function ($scope, idContainer, idContainerVersion, callback) {
+    tagManagerHelper.editVersion = function (idContainer, idContainerVersion, callback) {
```

Comment:
on line 84, there is another method, where the first parameter needs to be removed.
Otherwise the `create new variable` link on the `edit tag` page throws an error.